README.md

```diff
@@ -34,238 +76,656 @@ You can learn more about the Deepgram API at [developers.deepgram.com](https://d
+[See our API reference for more info](https://developers.deepgram.com/reference/speech-to-text-api/listen).
+
+[See the Example for more info](./examples/speech-to-text/rest/async/url/main.py).
+
+### Local Files (Asynchronous)
+
+Transcribe audio from a file.
 
-## Quickstarts
+```python
+from deepgram import PrerecordedOptions
+
+response = deepgram.listen.rest.v("1").transcribe_file_async(
+    source=open("path/to/your/audio.wav", "rb"),
+    callback_url="https://your-callback-url.com/webhook",
+    options=PrerecordedOptions(model="nova-3") # Apply other options
+)
+```
+
+[See our API reference for more info](https://developers.deepgram.com/reference/speech-to-text-api/listen).
 
-This SDK aims to reduce complexity and abtract/hide some internal Deepgram details that clients shouldn't need to know about.  However you can still tweak options and settings if you need.
+[See the Example for more info](./examples/speech-to-text/rest/async/file/main.py).
 
-### PreRecorded Audio Transcription Quickstart
+## Streaming Audio
 
-You can find a [walkthrough](https://developers.deepgram.com/docs/python-sdk-pre-recorded-transcription) on our documentation site. Transcribing Pre-Recorded Audio can be done using the following sample code:
+Transcribe streaming audio.
 
 ```python
-AUDIO_URL = {
-    "url": "https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav"
-}
+from deepgram import LiveOptions, LiveTranscriptionEvents
 
-## STEP 1 Create a Deepgram client using the API key from environment variables
-deepgram: DeepgramClient = DeepgramClient("", ClientOptionsFromEnv())
+# Create a websocket connection
+connection = deepgram.listen.websocket.v("1")
 
-## STEP 2 Call the transcribe_url method on the prerecorded class
-options: PrerecordedOptions = PrerecordedOptions(
-    model="nova-3",
-    smart_format=True,
+# Handle transcription events
+@connection.on(LiveTranscriptionEvents.Transcript)
+def handle_transcript(result):
+    print(result.channel.alternatives[0].transcript)
+
+# Start connection with streaming options
+connection.start(LiveOptions(model="nova-3", language="en-US"))
+
+# Send audio data
+connection.send(open("path/to/your/audio.wav", "rb").read())
+
+# Close when done
+connection.finish()
+```
+
+[See our API reference for more info](https://developers.deepgram.com/reference/streaming-api).
+
+[See the Examples for more info](./examples/speech-to-text/websocket/).
+
+## Transcribing to Captions
+
+Transcribe audio to captions.
+
+### WebVTT
+
+```python
+from deepgram_captions import DeepgramConverter, webvtt
+
+transcription = DeepgramConverter(dg_response)
+captions = webvtt(transcription)
+```
+
+### SRT
+
+```python
+from deepgram_captions import DeepgramConverter, srt
+
+transcription = DeepgramConverter(dg_response)
+captions = srt(transcription)
+```
+
+[See our stand alone captions library for more information.](https://github.com/deepgram/deepgram-python-captions).
```

Comment:
IMHO this section shouldn't be in the Python SDK, it's not part of the SDK's functionality.

Reply:
Valid point, but it's in the JS SDK, so I followed this pattern: https://github.com/deepgram/deepgram-js-sdk?tab=readme-ov-file#transcribing-to-captions

I think think @lukeocodes  probably did this to call attention to this library as it can be used with the SDK or as stand alone.

Let me know if that eases your concerns.